mage_ai/data_preparation/templates/data_exporters/bigquery.py

```diff
@@ -25,13 +26,14 @@ def export_data_to_big_query(df: DataFrame) -> None:
     ```
     BigQuery.with_credentials_object({'service_key': ...}, **kwargs)
     ```
+
+    Alternatively, all parameters can be specified in the configuration file.
     """
     table_id = 'your-project.your_dataset.your_table_name'
-    config = {
-        # Specify any other configuration settings here to pass to BigQuery client
-        'project': 'your_project_name',
-    }
-    return BigQuery(**config).export(
+    config_path = './default_repo/io_config.yaml'
```

Comment:
let's use `get_repo_path` from `mage_ai.data_preparation.repo_manager` to get the correct repo path.

Reply:
Fixed, templates now use `get_repo_path`